docs/commands.md

```diff
@@ -184,7 +185,17 @@ Installs source{d} Engine components images.
 
 ### srcd components start
 
-*status*: ❌ TBD
+Start source{d} Engine components.
+
+*arguments*:
+  * `component`: the name of the component image. It must be one of:
```

Comment:
I don't know how much implicit it is, but maybe we should say that also the dependencies of the components are started.

Reply:
updated